js/index.js

```diff
@@ -2,30 +2,57 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
   //... your code goes here
-}
+  let arrSumTot = [];
+  for (let i = 0; i < product.length; i++) {
+    const price = product[i].querySelector('.price span');
+    const quantity = product[i].querySelector('.quantity input');
+    const subtotal = product[i].querySelector('.subtotal span');
+    let priceNum = parseFloat(price.innerHTML);
+    let quantityNum = parseFloat(quantity.value);
+    let subTotalNum = priceNum * quantityNum;
+    arrSumTot.push(subTotalNum);
+    subtotal.innerHTML = `${subTotalNum}`
```

Comment:
No es necesario ya que estamos en un archivo js, recuerda que ocupamos `${} ` para hacer uso de expresiones de javascript en un string 
```suggestion
    subtotal.innerHTML = subTotalNum
```